### PR TITLE
Create 2018-08-07.md

### PR DESCRIPTION
**Pull request by jridgewell**

### CODE REVIEW

File: meetings/2018-08-07.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...

+        * Placing decorators before export prevents decorating the export itself.

+        * RB: I opened #135 to discuss decorating exports. If we decide it doesn’t really. It is difficult to take an argument that it prevents, if we don’t even know if we want to decorate exports.

+        * YK: The current syntax it not hostile to whether it’s before or after decorators.

+        * Diego: The semantics of the location would imply would mean.

+        * RB: I hate having decorators before and after the export (one decorating the export and one decorating the class). I don’t think this is viable.
```
Comment (rbuckton) [inline meetings/2018-08-07.md]: I don't think I used the word "hate". I am opposed to having both.

**Review by rbuckton — CHANGES_REQUESTED** [review]

**Review by rbuckton — APPROVED** [review]
Not sure if anyone else has other comments.

File: meetings/2018-08-07.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...

+        * RB: This allows us to decorate the export even if we put class decorators before export.

+        * BF: I think we argued this before, you have to have proposals to argue??? I think this should be the…

+        * DE: Thank you RB for making this gist.

+* toString definition https://github.com/tc39/proposal-decorators/issues/109#issuecomment-410855443 

+    * Decorators aren't included in methods' toString (and neither is static)
```
Comment (ljharb) [inline meetings/2018-08-07.md:65]: This isn't something I think we have consensus on.

File: meetings/2018-08-07.md [hunk 3]
```diff
@@ -0,0 +1,82 @@
...

+    * Dan's take: include static and decorators in method toString
```
Comment (ljharb) [inline meetings/2018-08-07.md:66]: ^ this is what i'd like to happen.

**Pull request merged**